### DISCUSSION TITLE
Questão TI21#8

### DISCUSSION THREAD

**Discussion by paulocafee** [paulowh/TI21#15] [Q&A] [answered]
Qual é a diferença entre null e undefined em JavaScript?

**Comment by erickgusson** [comment, marked as answer]
Null quer dizer que o valor é nulo, não tem valor
Já undefined é que ele não foi achado

EX: Uma string sem nada dentro seria Null, ele procurar uma string e não achar nada é undefined

**Reply by erickgusson** [reply]
Melhorando o exemplo
string nome = "" seria null
string nome; seria undefined

**Reply by Guilhlm** [reply]
null = vazio
undefined = n tem um valor definido
